packages/@angular/cli/commands/serve.ts

```diff
@@ -20,7 +20,8 @@ export interface ServeTaskOptions extends BuildOptions {
   host?: string;
   proxyConfig?: string;
   liveReload?: boolean;
-  liveReloadClient?: string;
```

Comment:
@filipesilva I would like to rename `--public-host` to `--public-ident`. It's confusing to have 2 host options, and In my opinion, an identifier is more close to what the thing actually does. Hope this is not too late for that ;)